netdisco/discoverables/webos_tv.py

```diff
@@ -8,8 +8,12 @@ class Discoverable(SSDPDiscoverable):
     """Add support for discovering LG WebOS TV devices."""
 
     def info_from_entry(self, entry):
-        """Return the most important info from a uPnP entry."""
-        return urlparse(entry.values['location']).hostname
+        """Get most important info, which is name, model and host."""
+        name = entry.description['device']['friendlyName']
+        model = entry.description['device']['modelName']
+        host = urlparse(entry.values['location']).hostname
+
+        return name, model, host
```

Comment:
@balloob any updates on this ?
PS: If I try to return a dict which I suppose should be {'name: name, 'model':model, 'host':host } I get errors (if that's a dict). I that what u were referring as dict ?